cilium/cmd/cgroups_list.go

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cilium/cilium/api/v1/models"
+	pkg "github.com/cilium/cilium/pkg/client"
+	"github.com/cilium/cilium/pkg/command"
+)
+
+var cgroupsListNoHeaders bool
+
+// cgroupsListCmd represents the cgroups_list command
+var cgroupsListCmd = &cobra.Command{
+	Use:     "list",
+	Aliases: []string{"ls"},
+	Short:   "List cgroups metadata",
+	Run: func(cmd *cobra.Command, args []string) {
+		listCgroups()
+	},
+}
+
+func init() {
+	cgroupsCmd.AddCommand(cgroupsListCmd)
+	cgroupsListCmd.Flags().BoolVar(&cgroupsListNoHeaders, "no-headers", false, "Do not print headers")
+	command.AddOutputOption(cgroupsListCmd)
+}
+
+func listCgroups() {
+	resp, err := client.Daemon.GetCgroupsDumpMetadata(nil)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%s\n", pkg.Hint(err))
+		os.Exit(1)
+	}
+
+	w := tabwriter.NewWriter(os.Stdout, 5, 0, 3, ' ', 0)
+	printMetadata(w, resp.Payload)
+}
+
+func printMetadata(w *tabwriter.Writer, metadata *models.CgroupsDumpMetadata) {
+	if metadata == nil {
+		fmt.Fprint(w, "Unable to retrieve cgroups metadata")
+		w.Flush()
+		os.Exit(1)
+	}
+
+	containerMetas := metadata.ContainerMetadatas
+	sort.Slice(containerMetas, func(i, j int) bool { return containerMetas[i].CgroupID < containerMetas[j].CgroupID })
```

Comment:
I looked at the endpoint list example, they sorted there. It would be easier to find some specific pods if they are sorted (after I'll do grouping by pod name instead of cgroup id). That said I don't have a very strong preference here however, so I can remove the sorting part, if it doesn't make much sense to you.

Reply:
I don't see any reason to sort the output, so let's drop it for now.

Reply:
Ok, removed